docs/data/sdks/typescript-node/migration.md

```diff
@@ -3,6 +3,158 @@ title: Node.JS SDK Migration Guide
 ---
 
+Amplitude new Node.js SDK (`@amplitude/analytics-node`) features a plugin architecture and built-in type definition. New Node.js SDK isn't backwards compatible with maintenance Node.js SDK `@amplitude/node`. 
+
+To migrate to `@amplitude/analytics-node`, update your dependencies and instrumentation.
+
+## Terminology
+
+* `@amplitude/node`: Maintenance Node.js SDK
+* `@amplitude/analytics-node`: New Node.js SDK
+
+## Dependency
+
+Uninstall `@amplitude/node` by either `npm uninstall @amplitude/analytics-node` or deleting your dependency list in package.json
+
+```diff
+{
+    "dependencies": {
+-    "@amplitude/node": "^1",
+    }
+}
+```
+
+Install `@amplitude/analytics-node` by `npm install @amplitude/analytics-node`.
+
+## Instrumentation
+
+New Node.js SDK offers an API to instrument events. To migrate to it, you need to update a few calls. The following sections detail which calls have changed.
+
+### Initialization
+
+The maintenance Node.js SDK only supports namespace import. The new Node.js SDK supports namespace import (`import * as amplitude from '@amplitude/analytics-node'`) and named import (`import { init } from '@amplitude/analytics-node'`) as well. We are using named import in the examples of new Node.js SDK in this documentation.
+
+To initialize the SDK, call `init()`, with the user ID and configuration parameters.
+
+```diff
+- import * as Amplitude from '@amplitude/node'
++ import { init } from '@amplitude/analytics-node';
+
+var options = {};
+- const client = Amplitude.init(AMPLITUDE_API_KEY, options);
++ init(API_KEY, options);
+```
+
+### Configuration
+
+The new Node.js SDK configuration comes in a different shape. Some configurations are no longer supported.
+
+|@amplitude/node|@amplitude/analytics-node|
+|-|-|
+| `debug` | `logLevel` set to WARN level|
+| `logLevel` | `logLevel` |
+| `maxCachedEvents` | `flushQueueSize` |
+| `retryTimeouts` | `flushMaxRetries` can only be set to a number instead of an array of number as in `retryTimeouts`
+| `optOut` | `optOut` |
+| `retryClass` | CUSTOMIZATION NOT SUPPORT. Retry logic is handled by new Node.js SDK|
+| `transportClass` | `transportProvider` |
+| `serverUrl` | `serverUrl` |
+| `uploadIntervalInSec` | `flushIntervalMillis` is in milliseconds while `uploadIntervalInSec` is in seconds|
+| `minIdLength` | `minIdLength` |
+| `requestTimeoutMillis` | NOT SUPPORT |
+| `onRetry` | CUSTOMIZATION NOT SUPPORT. Retry logic is handled by the new Node.js SDK |
+
+### Tracking events
+
+#### `logEvent()`
+
+The `logEvent()` API maps to `track()`.
+
+```diff
++ import { track } from '@amplitude/analytics-node';
+
+const eventProperties = {
+    buttonColor: 'primary',
+};
+
+- client.logEvent({
+-     event_type: 'Button Clicked',
+-     user_id: 'user@amplitude.com',
+-     event_properties: eventProperties
+- });
++ track('Button Clicked', eventProperties, {
++     user_id: 'user@amplitude.com',
++ });
```

Comment:
track method also supports the event object as a first parameter.

Reply:
Change it to track event object which is more similar as the old example